docs/content/tutorial/manual-restart.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Manually Restart PostgreSQL"
+date:
+draft: false
+weight: 105
+---
+
+There are times when you might need to manually restart PostgreSQL. This can be done utilizing PGO's rolling updates with minimal downtime by simply adding or updating a custom annotation for the cluster's instance sets `spec.metadata.annotations`, the Pod template will change for all instances and will trigger a rolling update for the cluster.
```

Comment:
:pick: 
```suggestion
There are times when you might need to manually restart PostgreSQL. This can be done, with minimal downtime, by using PGO's rolling update functionality. By simply adding or updating a custom annotation to the cluster instance-set's `spec.metadata.annotations` section, the Pod template will change for all instances and trigger a rolling update for the cluster. 
```